test/standalone/Soil/soil_bc.jl

```diff
@@ -198,3 +198,13 @@ for FT in (Float32, Float64)
         @test abs(flux_expected - flux_int) < eps(FT)
     end
 end
+
+
+@testset "Boundary vars" begin
+    # AtmosDrivenFluxBC method tested in `test/standalone/Soil/climate_drivers.jl`
+    # Currently no other methods exist
+    struct Foo <: Soil.AbstractSoilBC end
+    @test Soil.boundary_vars(Foo(), ClimaLSM.TopBoundary()) == ()
```

Comment:
should we test with `ClimaLSM.BottomBoundary()` as the second argument as well?

Reply:
I added it in!